configs/main.go

```diff
@@ -84,6 +85,11 @@ func New() *Configs {
 		rootConfigPartialPath = ".railway/dev-config.json"
 	}
 	rootConfigPath := path.Join(os.Getenv("HOME"), rootConfigPartialPath)
+	
+	if (runtime.GOOS == "windows") {
+		rootConfigPath = path.Join(os.Getenv("APPDATA"), rootConfigPartialPath)
+	}
```

Comment:
I think we may be able to use Go's built-in `os.UserHomeDir` to obtain the correct directory https://golang.org/pkg/os/#UserHomeDir

Reply:
updated the code to use `os.UserHomeDir`, functionality seemed the same, the config will just be in the `C:\Users\USERNAME\` directory now (vs code also uses this). Also did some very limited testing on Mac (just logging in, displaying a projects' env variables and stuff) and it seemed to work (the .railway folder was in the HOME directory like it's supposed to be)

Reply:
`os.UserHomeDir` returns `%USERPROFILE%` on Windows.  This is generally not the preferred place to install app data, that's what appdata is for.

It is personally a pet peeve of mine when applications put any files in my `%USERPROFILE%` instead of Appdata roaming.

Reply:
I personally would recommend using `os.UserConfigDir` which returns `%APPDATA%` on windows. and `$HOME\.config` on Unix.

Reply:
That is a good point. We've discussed this and are going to stick with the current implementation (USERPROFILE) for now since it is basically the equivalent of what we do on Unix at the moment. Then in the future, we can migrate both to `os.UserConfigDir`

Reply:
Moving to `os.UserConfigDir` right now would require migrating all existing users, which is out of scope of this PR.